packages/components/nodes/sequentialagents/ConditionAgent/ConditionAgent.ts

```diff
@@ -185,6 +187,42 @@ class ConditionAgent_SeqAgents implements INode {
                 additionalParams: true,
                 optional: true
             },
+            {
```

Comment:
thanks @jeanibarz ! can we increment the `this.version = 2.0` to `3.0`, and have the parameter order as the same in Agent/LLMNode:

1. System Prompt
2. Prepend Messages History
3. Conversation History
4. Human Prompt